src/test/java/uk/gov/hmcts/reform/orgrolemapping/controller/FeatureFlagControllerTest.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.reform.orgrolemapping.controller;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import uk.gov.hmcts.reform.orgrolemapping.data.FlagConfig;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.FlagRequest;
+import uk.gov.hmcts.reform.orgrolemapping.domain.service.PersistenceService;
+import uk.gov.hmcts.reform.orgrolemapping.util.PersistenceUtil;
+
+import java.util.Objects;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
We don't need @RunWith(MockitoJUnitRunner.class) as we are initialising the mock through MockitoAnnotations.initMocks(this);.

Reply:
Done